index.js

```diff
@@ -11,43 +11,26 @@ const api = require('./src/api');
 const server = require('./server.js');
 const helpers = require('./helpers');
 
-const numCPUs = require('os').cpus().length;
-
-let workerCount = 0;
+const WORKERS = process.env.WEB_CONCURRENCY || 1;
 
 /**
- * This creates node cluster workers for bot app
+ * @description This creates node cluster workers for bot app
  * Workers maximizes the CPU utility and ensure that the bot runs with less likelihood of a downtime
- * For more info, see: https://www.sitepoint.com/how-to-create-a-node-js-cluster-for-speeding-up-your-apps/
- * and https://medium.com/the-andela-way/scaling-out-with-node-clusters-1dca4a39a2a
+ * For more info, see: https://devcenter.heroku.com/articles/node-concurrency
  */
-if (cluster.isMaster) {
-  console.log(`Master ${process.pid} is running`);
-
-  // Creates as many workers as the number of available CPUs
-  while (workerCount <= numCPUs) {
-    cluster.fork();
-    workerCount += 1;
-  }
+throng({
+  workers: WORKERS,
+  lifetime: Infinity,
```

Comment:
This `Infinity` seems to be a variable from somewhere, can't seem t pinpoint where from. Please give me some context.

Reply:
It's a global numeric variable representing infinity. See the [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity) for more information.